variables.tf

```
@@ -18,6 +18,12 @@ variable "workspace_id" {
   type        = string
 }
 
+variable "local_authentication_disabled" {
+  description = "Disable local authentication to enable RBAC."
```

Boolean variables are often phrased as questions. Also, could be useful to specify what the difference is between having this disabled or enabled:

```suggestion
  description = "Should local authentication be disabled for this Application Insights component? If true, the \"Monitoring Metrics Publisher\" role is required for Microsoft Entra authentication."
```